go.mod

```diff
@@ -61,4 +62,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
+	lukechampine.com/blake3 v1.2.1 // indirect
```

Comment:
why is it stated as an indirect dependency? :thinking:

Reply:
Think it may become indirect when I install it and don't use it at that time. I ran tidy and it fixed it.